authors/urls.py

```diff
@@ -23,4 +23,5 @@
     path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication')),
     path('api/', include('authors.apps.profiles.urls', namespace='profiles')),
     path('api/', include(('authors.apps.articles.urls', 'articles'), namespace='articles')),
+    path('api/articles/', include(('authors.apps.comments.urls', 'comments'), namespace='comments')),
```

Comment:
What happens if you add this URL to the articles' urls.py?

Reply:
This has been implemented. Thank you for the suggestion.